### ISSUE TITLE
Missing colorbar_position for matplotlib backend?

### ISSUE THREAD

**Issue by ceball** [ioam/holoviews#1942]
E.g. trying to do something like

```
%%opts Image [colorbar=True colorbar_position='left']
```

I get
 
```
Unexpected plot option 'colorbar_position' for Image in loaded backend 'matplotlib'.
```

I think mpl has `location` and `orientation`.

**Comment by ceball** [comment]
`hv.__version__` is `1.8.4-57-g96ecb1a`

**Comment by philippjfr** [comment]
Getting the colorbar to position correctly consistently in matplotlib was quite painful so I'm not sure how easy this would be.

**Comment by ceball** [comment]
> Getting the colorbar to position correctly, consistently in matplotlib was quite painful

Pleased to hear then that it's easier than it used to be; pretty sure last time I tried (2011? 2012?) it was extremely painful ;)

The only reason I reported this issue is because I happened to notice it when I tried out the mpl backend to compare against bokeh for a separate colorbar issue.

Do you have a way to mark an issue such that it's clear you know about it but don't intend to fix any time soon (i.e. that if someone needs it to be fixed soon, they should do it themselves)?

**Comment by philippjfr** [comment]
>Do you have a way to mark an issue such that it's clear you know about it but don't intend to fix any time soon (i.e. that if someone needs it to be fixed soon, they should do it themselves)?

No, but we should reorganize our tags at some point, probably borrowing heavily from what bokeh does.

**Comment by jlstevens** [comment]
Good idea. A good time to do that would be after closing the milestone after the next release.
